layers/+email/mu4e/packages.el

```diff
@@ -26,10 +26,26 @@
         :mode mu4e-main-mode
         :bindings
         (kbd "j") 'mu4e~headers-jump-to-maildir)
-      (evilified-state-evilify-map mu4e-headers-mode-map
-        :mode mu4e-headers-mode)
-      (evilified-state-evilify-map mu4e-view-mode-map
-        :mode mu4e-view-mode)
+
+      (evilified-state-evilify-map
+       mu4e-headers-mode-map
+       :mode mu4e-headers-mode
+       :bindings
+       (kbd "J") (lambda ()
+                   (interactive)
+                   (mu4e-headers-mark-thread-using-markpair '(read))))
+
+      (evilified-state-evilify-map
+       mu4e-view-mode-map
+       :mode mu4e-view-mode
+       :bindings
+       (kbd "C-j") 'mu4e-view-headers-next
+       (kbd "C-k") 'mu4e-view-headers-prev
+       (kbd "J") (lambda ()
+                   (interactive)
+                   (mu4e~view-in-headers-context
```

Comment:
That could be a problem. Usually it's a bad idea to use private APIs. Maybe I should try to get this function included in mu4e upstream instead.